### ISSUE TITLE
Not work without pretrained model?

### ISSUE THREAD

**Issue by AI-Hunter** [uvipen/Yolo-v2-pytorch#22]
How can It work from scratch (without pretrained model)?
Thanks

**Comment by JamesYang568** [comment]
![image](https://user-images.githubusercontent.com/49474766/224591482-52d2db9b-89bc-4c32-9522-0d93c21af4d3.png)
I just annotate this part，and it seems work properly. But the VOC2012 loss I got is twice higher than the author.
